### ISSUE TITLE
"MUST implement the "scrolled-doc" presentation mode" in 4.4 of EPUB 3.3 RS

### ISSUE THREAD

**Issue by murata2makoto** [w3c/epub-specs#2133]
This requirement does not have any tests.

**Comment by iherman** [comment]
Correct. We will have to complete the test suite in the coming months. One of the jobs we will have to do is to make a systematic review of all MUST (and maybe SHOULD) statements to plan for new tests.

Also: there is a separate test repository: https://github.com/w3c/epub-tests. Would be good to collect these issues there; are you o.k. if I transfer this issue to that repository?

**Comment by murata2makoto** [comment]
> Would be good to collect these issues there; are you o.k. if I transfer this issue to that repository?

That's fine to me.
